CONTRIBUTING.rst

```diff
@@ -334,15 +334,18 @@ Versioning 2.0.0 (refs: https://semver.org/ ).
     new MAJOR version is released, the old ``MAJOR.MINOR`` branch will be
     deleted and replaced by an equivalent tag.
 
-``X.Y.Z``
-    Where ``X.Y.Z`` is the ``MAJOR.MINOR.PATCH`` release.  Only
-    backwards-compatible bug fixes are allowed. In Sphinx project, PATCH
-    version is used for urgent bug fix.
+A new MAJOR release can be expected every year, while MINOR releases are made
+roughly every three months. For example:
```

Comment:
FYI: We've try to release in every other months. So 2.4.0 will be released in February. And 3.0 will be April.

Reply:
You're saying the release cycle is every two months and not every three months? I can fix this if so.

Reply:
Yes, please.